src/tools/compiletest/src/runtest.rs

```diff
@@ -3124,7 +3125,12 @@ impl<'test> TestCx<'test> {
                     errors += self.compare_output("stdout", &normalized_stdout, &expected_stdout);
                 }
                 if !self.props.dont_check_compiler_stderr {
-                    errors += self.compare_output("stderr", &normalized_stderr, &expected_stderr);
+                    let kind = if self.props.stderr_per_bitwidth {
+                        format!("stderr.{}bit", size_of::<isize>() * 8)
```

Comment:
This may not necesarily be correct. I think you can test a 32bit x86 compiler with a 64bit x86 build system.

Reply:
Do we do that in our testsuite?